sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,328 @@
+- Added new FPGA loop attribute `enable_loop_pipelining`. [4147b8ded75f]
+- Started to print device aspects in `sycl-ls --verbose`. [e32ab1759fed]
+- Added tangle/opportunistic algorithms. [29e629e00e83]
+- Implemented `info::device::backend_version` query. [df743a5c97e1]
+- Added `fixed_size_group` support to algorithms. [01ac0338bfa9]
+- Added simple `abs(int)` to imf libdevice. [2a3aaeea5be8]
+- Added `ballot_group` support to algorithms. [70a35de4eacb]
+- Implemented interoperability support for images for the Level Zero backend. [836ceec36b57]
+- Added marray support for math built-in functions. [a7a6de2e40a2][c274f40588ff][2953d2548791]
+- Enabled inline assembly support in ESIMD kernels. [f5a062dd057f][48abe38c0bee]
+- Enhanced interop queue interface to choose standard or immediate commandlists. [0ff3ec40e241]
+- Enabled double type for `atomic_update()` ESIMD API. [98c443543c72]
+- Added support for `addc` and `subb` operations for ESIMD. [0a098933d9ba]
+
+### Documentation
+- Added experimental SYCL bindless images extension [`sycl_ext_oneapi_bindless_images`](doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc) proposal.
+- Added [`sycl_ext_oneapi_work_group_local`](doc/extensions/proposed/sycl_ext_oneapi_work_group_local.asciidoc) extension proposal. [1d6cff1a8e5a]
+- Moved [sycl_ext_oneapi_device_global](doc/extensions/experimental/sycl_ext_oneapi_device_global.asciidoc) from proposed to experimental. [d812d1e13ab5]
+- Added [`DeviceConfigFile`](doc/design/DeviceConfigFile.md) design document. [3f4b778dd163]
+- Added [`sycl_ext_oneapi_address_cast`](doc/extensions/proposed/sycl_ext_oneapi_address_cast.asciidoc) extension. [9cab5598bc82]
+- Added [`sycl_ext_intel_grf_size`](doc/extensions/experimental/sycl_ext_intel_grf_size.asciidoc) extension. [83d099758770]
+- Added proposal for [`sycl_ext_oneapi_append_and_shift`](doc/extensions/proposed/sycl_ext_oneapi_append_and_shift.asciidoc) extension. [1a283acaac3c]
+- Updated description of accessor-based memory APIs in [`sycl_ext_intel_esimd`](sycl/doc/extensions/experimental/sycl_ext_intel_esimd/sycl_ext_intel_esimd.md). [9e5889918277]
+- Added draft of [`sycl_ext_oneapi_prefetch`](sycl/doc/extensions/proposed/sycl_ext_oneapi_prefetch.asciidoc). [034ac0958662]
+- Added draft of [`sycl_ext_oneapi_barrier`](doc/extensions/proposed/sycl_ext_oneapi_barrier.asciidoc) [32144edcf982]
+- Added legacy SYCL 1.2.1 image aspect. [78b709dd5bb8]
+- Added [`sycl_ext_intel_data_flow_pipes_properties`](sycl/doc/extensions/experimental/sycl_ext_intel_data_flow_pipes_properties.asciidoc) extension. [5b0a4615874b]
+- Updated [`sycl_ext_intel_dataflow_pipes`](doc/extensions/supported/sycl_ext_intel_dataflow_pipes.asciidoc) extension for FPGA host. [6b2d66b8d52f]
+
+## Improvements
+
+### SYCL Compiler
+- Started to use LLVM IR instead of GenX intrinsics for ESIMD block loads/stores - USM and SLM - and group/thread id/size queries. [8af94a01f063][7e7971a57666]
```

Comment:
This is an internal thing, no effect on user, probably better to keep off release notes. Tagging @v-klochkov and @sarnex

Reply:
I agree, removed.